src/java/org/apache/cassandra/tools/nodetool/GossipInfo.java

```diff
@@ -18,16 +18,20 @@
 package org.apache.cassandra.tools.nodetool;
 
 import io.airlift.airline.Command;
+import io.airlift.airline.Option;
 
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
 
 @Command(name = "gossipinfo", description = "Shows the gossip information for the cluster")
 public class GossipInfo extends NodeToolCmd
 {
+    @Option(title = "resolve_ip", name = {"-r", "--resolve-ip"}, description = "Show node domain names instead of IPs")
```

Comment:
should not this be `Show node's domain names insteao of IPs` ? You can reformulate to "Show domain names of a node instead of its IPs".

Does a node have multiple IP? If it can have just one, plural in "IPs" should be changed to singular.

Reply:
@smiklosovic , I've basically copy-pasted the existing phrasing for `--resolve-ip` options at:

- https://cassandra.apache.org/doc/4.1/cassandra/tools/nodetool/status.html
- https://cassandra.apache.org/doc/4.1/cassandra/tools/nodetool/ring.html

I think the intention was that commands will output data about multiple nodes in production.
And IPs for these nodes will be resolved to their relevant domain names.

Should I still commit a singular phrasing? If yes, should I re-phrase along the way the `status` and `ring` commands to align all three of them to the same `--resolve-ip` option description?